### PR TITLE
Testing PHP resource server

### PR DESCRIPTION
**Pull request by vijetmahabaleshwar-okta**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,3 +14,7 @@ script:
 
 before_install:
   - "npm install -g npm@5.6.0"
+  - sudo apt-add-repository ppa:ondrej/php -y
+  - sudo apt-get update
+  - sudo apt-get install curl php7.0
```
Comment (vijetmahabaleshwar-okta) [inline .travis.yml]: @bretterer - Can you take a look at this? I'm trying to install php 7 on travis but after running these commands, I still get the following error - 
`This package requires php ^7.0 but your PHP version (5.6.32) does not satisfy that requirement.`
Do I have to uninstall php5 before this?

**Comment by robertjd** [comment]
Are we intending to merge this, or is this just a demo branch?

**Comment by bretterer** [comment]
We were talking offline... considering making a set of tests that mocks the resource server, so the library does not rely on any other library resource servers

**Comment by robertjd** [comment]
If we're going to build a resource server mock we should extend the functionality that already exists in the TCK, using the wiremock suite we built

**Comment by bretterer** [comment]
Agreed... I have talked with @vijetmahabaleshwar-okta about it some and will figure out where to go with it

**Comment by vijetmahabaleshwar-okta** [comment]
Closing this PR as we don't want to introduce more dependencies for testing the resource server.

**Pull request closed without merging**